src/org/parosproxy/paros/extension/manualrequest/http/impl/ManualHttpRequestEditorDialog.java

```diff
@@ -566,10 +567,15 @@ private JSplitPane createSplitPane(int orientation) {
 	}
 	
 	public void addPersistentConnectionListener(PersistentConnectionListener listener) {
-		((HttpPanelSender) getMessageSender()).addPersistentConnectionListener(listener);
+		(getMessageSender()).addPersistentConnectionListener(listener);
```

Comment:
Could be removed the surrounding parentheses too. (Same comment for the following change.)